spec/controllers/tasks_controller_spec.rb

```diff
@@ -0,0 +1,289 @@
+require 'rails_helper'
+
+describe TasksController, type: :controller do
+  let(:error_response) { { error: 'Something bad happen. Try again later.' }.stringify_keys }
```

Comment:
1. .stringify_keys is not needed here. If you need your hash to have string keys, just write them as string -- `{ 'error' => 'Something bad happen. Try again later.' }`
2. let's move this `let` to every context, where it is used, because we don't need any excess vars to be available in other contexts